libc/src/__support/FPUtil/FEnvImpl.h

```diff
@@ -28,7 +28,7 @@
 // the apple condition here should be removed.
 #elif defined(LIBC_TARGET_ARCH_IS_X86) && !defined(__APPLE__)
 #include "x86_64/FEnvImpl.h"
-#elif defined(LIBC_TARGET_ARCH_IS_ARM)
+#elif defined(LIBC_TARGET_ARCH_IS_ARM) && __has_builtin(__builtin_arm_get_fpscr)
```

Comment:
iirc `__has_builtin` is a clang specific extension, should we also check what compiler we're using?

Reply:
gcc also has `__has_builtin` : https://gcc.gnu.org/onlinedocs/cpp/_005f_005fhas_005fbuiltin.html
But I'm switching to use our wrapper macro `LIBC_HAS_BUILTIN` instead.